prime-router/src/main/kotlin/Report.kt

```diff
@@ -819,12 +819,19 @@ class Report : Logging {
                     it.patientTribalCitizenshipCode = row.getStringOrNull("patient_tribal_citizenship_code")
                     it.patientPreferredLanguage = row.getStringOrNull("patient_preferred_language")
                     it.patientNationality = row.getStringOrNull("patient_nationality")
+                    it.patientSpecies = row.getStringOrNull("patient_species")
+                    it.patientSpeciesCode = row.getStringOrNull("patient_species_code")
 
                     it.reasonForStudy = row.getStringOrNull("reason_for_study_text")
                     it.reasonForStudyCode = row.getStringOrNull("reason_for_study_id")
 
-                    it.testResultCode = row.getStringOrNull("test_result_text").trimToNull()
-                    it.testResult = row.getStringOrNull("test_result_id").trimToNull()
+                    it.testResultCode = row.getStringOrNull("test_result_id").trimToNull()
+                    it.testResult = row.getStringOrNull("test_result_text").trimToNull()
+                    it.testResultNormalized = if (it.testResultCode != null) {
+                        metadata.findValueSet("monkeypox/test_result")?.toDisplayFromCode(it.testResultCode)
+                    } else {
+                        null
+                    }
```

Comment:
If the test result code has a value, then look up the monkeypox valueset and get the display value so we have a normalized value.